packages/react-server/src/ReactFizzServer.js

```diff
@@ -3794,12 +3794,22 @@ function abortTask(task: Task, request: Request, error: mixed): void {
           error.$$typeof === REACT_POSTPONE_TYPE
         ) {
           const postponeInstance: Postpone = (error: any);
-          const fatal = new Error(
-            'The render was aborted with postpone when the shell is incomplete. Reason: ' +
-              postponeInstance.message,
-          );
-          logRecoverableError(request, fatal, errorInfo, null);
-          fatalError(request, fatal, errorInfo, null);
+          const trackedPostpones = request.trackedPostpones;
+
+          if (trackedPostpones !== null && segment !== null) {
+            // We are prerendering. We don't want to fatal when the shell postpones
+            // we just need to mark it as postponed.
+            logPostpone(request, postponeInstance.message, errorInfo, null);
+            trackPostpone(request, trackedPostpones, task, segment);
+            finishedTask(request, null, segment);
+          } else {
```

Comment:
The factoring here isn't great but I don't want to completely refactor this code. I have to recheck wether the segment actually exists to mark it as postponable but if we're tracking postpones it would definitely exist. But mabye I should